azext_iot/iothub/providers/helpers/state_constants.py

```diff
@@ -0,0 +1,46 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
```

Comment:
These are all user facing strings, `state_strings.py` is a more appropriate file name